common/reviews/api/core.api.md

```diff
@@ -1212,6 +1257,8 @@ interface IImage {
     // (undocumented)
     height: number;
     // (undocumented)
+    imageFrame?: any;

```

Comment:
Please ensure this is documented as being 1 based, since it is sometimes hard to remember which one is which.

Reply:
it is the imageFrame object from the CSWIL